packages/sit-onyx/src/components/OnyxIcon/OnyxIcon.vue

```diff
@@ -0,0 +1,93 @@
+<script lang="ts" setup>
+import type { OnyxIconProps } from "./types";
+
+const props = withDefaults(defineProps<OnyxIconProps>(), {
+  size: "sm",
+  color: "currentColor",
+});
+</script>
+
+<template>
+  <!-- eslint-disable vue/no-v-html -->
+  <i
```

Comment:
why `<i>`?

Reply:
We need a wrapper component due to the `v-html` since we need to put it somewhere :D Google said that `<i>` and `<span>` are often used for icons: https://www.w3schools.com/icons/#:~:text=The%20and%20%3Cspan,widely%20used%20to%20add%20icons.

Should I change it to span or something else? I don't have a preference here

Reply:
Maybe `figure`, but it is not an inline element

Reply:
@JoCa96 Should I change it to `<figure>` then and make it `display: inline-block`?
I would also be fine with just keeping `<i>` since it kinda sounds like "icon"

Reply:
the thing is, it does not really fit the [description](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i?retiredLocale=de) of what the i tag is

![image](https://github.com/SchwarzIT/onyx/assets/151019977/a9f75224-49e9-4de5-88ea-2bc8bed52fb8)

Reply:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/e37e09cf-bc0f-4889-b4a5-b358b0b66e0a)
figure sounds more like it :)
but span would also be fine for me ¯\\_(ツ)_/¯

Reply:
I changed it to `<figure>` :)